Ginger/GingerCore/Environments/Database.cs

```diff
@@ -220,6 +220,76 @@ public static List<string> DbTypes
             }
         }
 
+        public string NameBeforeEdit;
+
+        public static void UpdateDatabaseNameChangeInItem(object item, string prevVarName, string newVarName, ref bool namechange)
+        {
+            var properties = item.GetType().GetMembers().Where(x => x.MemberType == MemberTypes.Property || x.MemberType == MemberTypes.Field);
+            foreach (MemberInfo mi in properties)
```

Comment:
Verify if we need all this reflection code. 
We can directly update appName in the action itself